### ISSUE TITLE
Remote access no longer possible

### ISSUE THREAD

**Issue by danir-de** [hassio-addons/addon-unifi#427]
# Problem/Motivation

The Unifi network controlled by the add-on always shows as offline both in the UniFi app, as well as online on unifi.ui.com.

## Expected behavior

Network should be online, when the Add-on is running.

## Actual behavior

The network is being shown as offline.

## Steps to reproduce

- Install the UniFi Addon on ~Hass.io~ Home Assistant OS on as Raspberry Pi 4B.
- Check the Add-on settings. It states:
   >   Remote Access requires 64-bit Java Runtime Environment
UniFi Network requires a 64-bit Java runtime environment (JRE) to enable Remote Access from [unifi.ui.com.](https://unifi.ui.com/) You can download it from [www.java.com.](https://java.com/)

## Proposed changes

Include 64-Bit java package inside the container image.

**Comment by frenck** [comment]
"Hass.io" That term hasn't been used for years anymore, and is ambiguous.

This seems like an odd message, especially since the add-on is only shipped as 64 bits (there aren't 32 bits versions of the add-on available). I have no direct reason to see why it says that tbh. That said, the add-on runs on OpenJDK, maybe that is a support thing @ UniFi.

../Frenck

**Comment by danir-de** [comment]
I could fix it by restarting the whole HomeAssistant Supervisor, effectively rebooting the hardware device.

TLDR: Don't just restart the container/app. Restart the host system to get it working again.

Maybe someone finds this useful in the future.
